libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Identity/SFIdentityCoordinator.m

```diff
@@ -38,8 +38,8 @@
 
 // Private constants
 
-static NSString * const kHttpHeaderAuthorization             = @"Authorization";
-static NSString * const kHttpAuthHeaderFormatString          = @"Bearer %@";
+NSString * const kHttpHeaderAuthorization             = @"Authorization";
```

Comment:
Revert this change since we're switching to `SFRestAPI`.

Reply:
@bhariharan as we discussed offline, we will continue to use SFNetwork. No need for refresh since this call is made during auth completion.